client/config/webpack.config.common.js

```diff
@@ -6,7 +6,7 @@ const isInDebugMode = debugString => debugString === 'TRUE';
 const DEBUG = JSON.stringify(process.env.DEBUG);
 
 module.exports = {
-  entry: '../index.js',
+  entry: ['babel-polyfill', '../index.js'],
```

Comment:
Why do we need a babel-polyfill?

Reply:
@BolajiOlajide It allows the use of async-await.

Reply:
We should have `async.. await` out of the box... What node version are you using? @tosmak16

Reply:
@BolajiOlajide `v10.5.0`

Reply:
@BolajiOlajide Please try it from your end without babel-polyfill.

Reply:
Okay. Can you install this from `npm`: `babel-preset-latest-node` and update the `.babelrc` with this instead.

```{
  "presets": [
    ["env", {
      "targets": {
        "browsers": [
          "last 2 versions"
        ]
      }
    }],
    "stage-0",
    "react",
    "["latest-node", { "target": "current" }]"
  ]
}```

Reply:
Once that is done, remove the `polyfill` and try the async await again.

Let me know if it fails.

Reply:
@BolajiOlajide Alright... On it.